### PR TITLE
88x31 banner scrapper

### PR DESCRIPTION
**Pull request by rudy3333**
added a scrapper for https://cyber.dabamos.de/88x31/index.html

### CODE REVIEW

**Comment by rudy3333** [comment]
@nikhil25803 could you please take a look? <3

**Comment by rudy3333** [comment]
Will do, it's my first time working on a Library so I got to learn a lot!!

**Comment by rudy3333** [comment]
@nikhil25803 g2g?

File: src/scrape_up/banner88x31/scraper88x31.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            'https://cyber.dabamos.de/88x31/index4.html',
+            'https://cyber.dabamos.de/88x31/index5.html'
+        ]
+
+    def get_all(self):
+        """
+        Fetches all 88x31 banners
+        Returns:
+        Returns the list of all available 88x31 banners
+        """
```
Comment (nikhil25803) [inline src/scrape_up/banner88x31/scraper88x31.py:38]: This is not a proper way to write docstring, check other modules like `IMDB` or `GitHub`, and you'll get an idea about it.

Make this change, then I'll test it locally once and merge if every thing works fine.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Review by nikhil25803 — APPROVED** [review]
Looks good :))

**Comment by nikhil25803** [comment]
Great work @rudy3333 !!
Do not forget to
+ Share your contribution on socials, tag us, and Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Pull request merged**